lib/ReactViews/FeatureInfo/feature-info-download.scss

```diff
@@ -12,17 +12,21 @@
 }
 
 .icon--download {
-  display: inline-block;
-  vertical-align: bottom;
-  svg{
+  // the default icon styling for a dropdown is for a btn--primary, so we have to make some adjustments
+  padding-right: 10px;
+  svg {
+    position: absolute;
+    top: 6px;
+    right: 5px;
     height: 20px;
     width: 20px;
-    fill: $text-light;
+    fill: currentColor;
   }
 }
 
 .dropdown__list {
   display: flex;
+  flex-direction: column;
```

Comment:
<img width="230" alt="Screen Shot 2019-04-24 at 10 47 00 am" src="https://user-images.githubusercontent.com/7508939/56624947-5933f300-667e-11e9-963b-23deb41b4213.png">
I can't seem to see the change? Am I looking at the wrong place? If so, these drop-downs should probably have the same styles too :).

Reply:
Different dropdown. Nice catch :) I'll fix this one too.

The one I changed was the one that says "Download this Table".